### PR TITLE
Promoting the SDRS alert to Warning /Critical

### PR DESCRIPTION
**Pull request by rezash84**
Promoting the SDRS alert to Warning /Critical status and removed the info alert .

### CODE REVIEW

**Comment by christopherhans** [comment]
Did you encountered the alert already and used the linked playbook?

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 1]
```diff
@@ -258,16 +258,32 @@ groups:
...
+      description: "DataStore Cluster {{ $labels.storagepod }} utilization > 85%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
+      summary: "DataStore Cluster {{ $labels.storagepod }} utilization > 85%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
+
+- alert: SDRSClusterCapacityCritical
     expr:  vrops_storagepod_capacity_remaining_percentage <= 10
     for: 20m
     labels:
-      severity: info
+      severity: critical
       tier: vmware
```
Comment (christopherhans) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:281]:
```suggestion
      tier: vmware
      support_group: compute
```

Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/datastore.alerts:281]: There is a new label for the upcoming target operation model.

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 2]
```diff
@@ -258,16 +258,32 @@ groups:
       description: "Datastore `{{ $labels.datastore }}` has hosts that have lost all of the redundant paths to a storage device. ({{ $labels.vcenter }})"
       summary: "Datastore `{{ $labels.datastore }}` has hosts that have lost all of the redundant paths to a storage device. ({{ $labels.vcenter }})"
       
-  - alert: SDRSClusterCapacity
+ - alert: SDRSClusterCapacityWarning
+    expr:  vrops_storagepod_capacity_remaining_percentage <= 15
+    for: 20m
+    labels:
+      severity: warning
+      tier: vmware
```
Comment (christopherhans) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:266]:
```suggestion
      tier: vmware
      support_group: compute
```

**Review by christopherhans — CHANGES_REQUESTED** [review]

**Comment by rezash84** [comment]
Playbook have been created and reviewed  for this alert  covers both Warning and Critical alert instances  and this is to replace the old EPH datastore capacity alerts we have in place currently.
Moreover , the SDRS alert have been in observation period for 3 weeks to ensure correct functionality .
Please review and advise .
Thanks

**Comment by christopherhans** [comment]
@artherd42 asking for your feedback if this alert can be rolled out directly from info to warning&critical.
See Rezas comment above.

**Comment by artherd42** [comment]
@rezash84 Did you see false-positive alerts during the 3 observation weeks? And got these alerts fixed? Did the ops team has accepted the new alert?

**Comment by rezash84** [comment]
Hi @artherd42 , yes we have fixed  issue related to the alert during observation period and the Ops team has accepted the new alerts as well .Thank you!

**Review by artherd42 — APPROVED** [review]
lgtm

**Pull request merged**